src/main/scala/nl.knaw.dans.easy.bagstore/command/CommandLineOptionsComponent.scala

```diff
@@ -118,9 +118,9 @@ trait CommandLineOptionsComponent {
 
     val stream = new Subcommand("stream") {
       descr("Retrieves an item by streaming it to the standard output")
-      val format: ScallopOption[ArchiveStreamType] = opt(name = "format",
+      val format: ScallopOption[ArchiveStreamType] = opt(name = "format", short = 'f',
         descr = "stream item packaged in this format (tar|zip)")
-      val forceInactive: ScallopOption[Boolean] = opt(name = "force-inactive",
+      val forceInactive: ScallopOption[Boolean] = opt(name = "force-inactive", //TODO no short since f is reserved for format?
```

Comment:
This is confusing me at the moment. I would expect a change in a readme too. Ah that check should be applied to `01_manual.html`. 

Make sure each short has only one meaning in each sub-command, preferably even the same meaning in all subcommands when the same options happens in multiple subcommands. Do we need backward compatibility in other words do we have to adjust some scripts somewhere? I didn't check any specs so far on this subject.